code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -437,7 +437,8 @@
 	multi_sprite_step = AMMO_BOX_MULTI_SPRITE_STEP_ON_OFF
 
 /obj/item/ammo_box/magazine/m556/arg
-	name = "\improper ARG magazine (5.56mm)"
+	name = "\improper M26 STNEQ magazine (5.56mm)"
```

Comment:
"STNEQ" is an orphan acronym that is not explained anywhere. Dropping it directly into Google fails to demystify it.

The actual rifle that it goes into is simply called the M26A2 assault rifle, so this also does not help identify the meaning.